Cargo.toml

```diff
@@ -52,6 +52,7 @@ cosmwasm-schema = "1.2.1"
 apollo-cw-asset = "0.1.0"
 cw-controllers = "1.0.1"
 cw-dex = "0.5.1"
+cw-dex-astroport  = { version = "0.2.0-rc.1", optional = true}
```

Comment:
You'll also have to stop using Pool enum from `cw-dex` on https://github.com/apollodao/cw-dex-router/blob/master/src/operations.rs#L7

and create your own pool enum with variants based on feature flags in this crate.